lib/flatpak-remote.h

```diff
@@ -25,6 +25,15 @@
 #ifndef __FLATPAK_REMOTE_H__
 #define __FLATPAK_REMOTE_H__
 
+#ifdef FLATPAK_ENABLE_P2P
+typedef enum
```

Comment:
Fixed and needing review in https://github.com/flatpak/flatpak/pull/884/commits/eddd2082919399204c0ab67308cc92bed8b15724.